src/components/ConfigProvider/ConfigProvider.stories.tsx

```diff
@@ -9,6 +9,7 @@ import { Tab, Tabs, TabVariant } from '../Tabs';
 import { IconName } from '../Icon';
 import { CompactPicker } from 'react-color';
 import { ConfigProvider, OcThemeNames, useConfig } from './';
+import { MatchScore } from '../MatchScore/MatchScore';
```

Comment:
Can you create an index.ts and export the component from that.
This import will be cleaner then. Also export it in octuple.js